src/documents/components/FileDetail/FileDetail.js

```diff
@@ -103,6 +104,22 @@ const ActionButtons = ({
           />
         }
       />
+      <Popup
```

Comment:
Not all file types will have an extract config, so we need to perhaps add a key in the `fileTypeDetail` enum for that and then only display this button for those valid file types.